Ryujinx.HLE/HOS/Horizon.cs

```diff
@@ -357,29 +379,29 @@ public void LoadNsp(string nspFile)
 
             PartitionFileSystem nsp = new PartitionFileSystem(file.AsStorage());
 
-            foreach (DirectoryEntry ticketEntry in nsp.EnumerateEntries("*.tik"))
+            foreach (DirectoryEntryEx ticketEntry in nsp.EnumerateEntries("/", "*.tik"))
             {
-                Ticket ticket = new Ticket(nsp.OpenFile(ticketEntry.FullPath, OpenMode.Read).AsStream());
+                Result rc = nsp.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```

Comment:
```suggestion
                Result resultCode = nsp.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```
and add an empty line after.